app/unite-framework/unitelist.ts

```diff
@@ -1,14 +1,14 @@
 import {Injectable, Inject} from 'angular2/core';
-import {Page, Loading, NavController, NavParams} from 'ionic-angular';
+import {Page, Loading, NavController, NavParams, Toast} from 'ionic-angular';
 import {Http} from 'angular2/http';
 import 'rxjs/add/operator/map';
 
 @Injectable()
 export class UniteList {
   selectedItem: any;
   icons: string[];
-  items: Array<{title: string, note: string, icon: string}>;
-  url: string;
+  items: Array<{ title: string, note: string, icon: string }>;
```

Comment:
@manishks12288 this is still not right. Need to add all supported variables for a menu. Missing parent, children, order, component and a few more